package.json

```diff
@@ -2,7 +2,7 @@
   "name": "brown-university-components",
   "description": "Components for building React-based applications at Brown University",
   "author": "Colin Cummings <colin_cummings@brown.edu>",
-  "version": "1.7.0",
+  "version": "1.8.0",
```

Comment:
I don't think we need a version bump for these changes since they don't affect package consumers. These are just build tool changes (all of the changes are in `devDependencies`).